org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/typeHierarchy/TypeHierarchyViewContentProvider.java

```diff
@@ -90,14 +89,12 @@ public boolean hasChildren(Object element) {
 	public void inputChanged(final Viewer viewer, final Object oldInput, final Object newInput) {
 		ITreeContentProvider.super.inputChanged(viewer, oldInput, newInput);
 
-		treeViewer = (TreeViewer) viewer;
 		if (newInput instanceof HierarchyViewInput viewInput) {
-			hierarchyItems = null;
```

Comment:
shouldn't this be  `hierarchyItems = Collections.emptyList();` now?

Reply:
I removed this line, because the shown type hierarchy and its members should not be deleted if there is no Type Hierarchy for the current selection (i.e. select an integer variable and press F4 -> Type Hierarchy view does not change (Behavior of the old Type Hierarchy view))